packages/composites/multi-address-list/test/integration/__snapshots__/adapter.test.ts.snap

```diff
@@ -0,0 +1,61 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`execute address-list endpoint should return success 1`] = `
```

Comment:
This doesn't feel representative of the request that would be made from proof-of-reserves. The `network` and `chainId` at least should be the same across all feeds (since the `proof-of-reserves` input params would be passed through to each)

Shouldn't we fail when we receive different networks from the different custodians?

Reply:
I think this is a fixture issue, but you are right, currently the composite EA doesn't do any validation. In theory prod source EAs should not return assets for different network,  as long as the input data sent to them is correct. I agree that we can add some checks here for `chainId` and `network`.

Reply:
Fixed. chainId and network are not top level EA params